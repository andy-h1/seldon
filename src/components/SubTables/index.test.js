import React from 'react';
import { render } from '@testing-library/react';
import { SubTables } from '.';

describe('Sub Tables component', () => {
  it('should render', () => {
    const { container } = render(<SubTables />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
