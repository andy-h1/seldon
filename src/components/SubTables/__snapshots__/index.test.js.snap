// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sub Tables component should render 1`] = `
<div
  class="sc-gsTCUz EZDVl"
>
  <div
    class="MuiTableContainer-root"
  >
    <table
      aria-label="sum table"
      class="MuiTable-root makeStyles-root-1"
    >
      <thead
        class="MuiTableHead-root"
      >
        <tr
          class="MuiTableRow-root makeStyles-header-3 MuiTableRow-head"
        >
          <th
            class="MuiTableCell-root MuiTableCell-head makeStyles-headerText-4 MuiTableCell-alignLeft MuiTableCell-sizeSmall"
            scope="col"
          >
            Type of Action
          </th>
          <th
            class="MuiTableCell-root MuiTableCell-head makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            scope="col"
          >
            Value One
          </th>
          <th
            class="MuiTableCell-root MuiTableCell-head makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            scope="col"
          >
            Value Two
          </th>
        </tr>
      </thead>
      <tbody
        class="MuiTableBody-root"
      >
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-text-6 MuiTableCell-alignLeft MuiTableCell-sizeSmall"
          >
            Videos played
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            10
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            10
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-text-6 MuiTableCell-alignLeft MuiTableCell-sizeSmall"
          >
            Watch finished
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            100
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            1000
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-text-6 MuiTableCell-alignLeft MuiTableCell-sizeSmall"
          >
            Comment posted
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            100
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            100
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-text-6 MuiTableCell-alignLeft MuiTableCell-sizeSmall"
          >
            Likes gained
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            1000
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-headerText-4 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            10000
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-totalText-7 MuiTableCell-alignLeft MuiTableCell-sizeSmall"
          >
            Total
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-totalText-7 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            1210
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body makeStyles-totalText-7 MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            11110
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <div
    class="makeStyles-root-8"
  >
    <div
      class="MuiTableContainer-root"
    >
      <table
        aria-label="sub-table "
        aria-labelledby="tableTitle"
        class="MuiTable-root makeStyles-table-10"
      >
        <thead
          class="MuiTableHead-root makeStyles-header-11"
        >
          <tr
            class="MuiTableRow-root makeStyles-row-13 MuiTableRow-head"
          >
            <th
              class="MuiTableCell-root MuiTableCell-head makeStyles-headerText-12 MuiTableCell-alignLeft MuiTableCell-paddingNone MuiTableCell-sizeSmall"
              scope="col"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-16"
                role="button"
                tabindex="0"
              >
                Last Purchased
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                  />
                </svg>
              </span>
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head makeStyles-headerText-12 MuiTableCell-alignRight MuiTableCell-sizeSmall"
              scope="col"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-16"
                role="button"
                tabindex="0"
              >
                When
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                  />
                </svg>
              </span>
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head makeStyles-headerText-12 MuiTableCell-alignRight MuiTableCell-sizeSmall"
              scope="col"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiTableSortLabel-root makeStyles-labelText-16"
                role="button"
                tabindex="0"
              >
                Earned
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                  />
                </svg>
              </span>
            </th>
          </tr>
        </thead>
        <tbody
          class="MuiTableBody-root"
        >
          <tr
            class="MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignLeft MuiTableCell-paddingNone MuiTableCell-sizeSmall"
              role="cell"
              scope="row"
            >
              Mockups for Figma
            </th>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              16 mins
            </td>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-earnedText-15 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              $24.40
            </td>
          </tr>
          <tr
            class="MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignLeft MuiTableCell-paddingNone MuiTableCell-sizeSmall"
              role="cell"
              scope="row"
            >
              Material Icons Set
            </th>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              2 hours
            </td>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-earnedText-15 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              $14.40
            </td>
          </tr>
          <tr
            class="MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignLeft MuiTableCell-paddingNone MuiTableCell-sizeSmall"
              role="cell"
              scope="row"
            >
              Material Design System
            </th>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              1 days
            </td>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-earnedText-15 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              $68.00
            </td>
          </tr>
          <tr
            class="MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignLeft MuiTableCell-paddingNone MuiTableCell-sizeSmall"
              role="cell"
              scope="row"
            >
              Material Desktop System
            </th>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              1 days
            </td>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-earnedText-15 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              $128.50
            </td>
          </tr>
          <tr
            class="MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignLeft MuiTableCell-paddingNone MuiTableCell-sizeSmall"
              role="cell"
              scope="row"
            >
              iOS toolkit for Figma
            </th>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-text-14 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              2 days
            </td>
            <td
              class="MuiTableCell-root MuiTableCell-body makeStyles-earnedText-15 MuiTableCell-alignRight MuiTableCell-sizeSmall"
            >
              $58.00
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
